<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>widget</class>
 <widget class="QWidget" name="widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1220</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1220</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1220</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>我的音乐</string>
  </property>
  <property name="windowIcon">
   <iconset theme="icondesktop">
    <normaloff>image/icondesktop_32dp.png</normaloff>image/icondesktop_32dp.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{

background-repeat:no-repeat;



}</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1200</width>
     <height>700</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>1200</width>
     <height>700</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1200</width>
     <height>700</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QFarme#frame{
border-radius:5px;
border-image: url(:/image/image/background/bg1.jpg);
}
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <widget class="QSlider" name="horizontalSlider">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>650</y>
      <width>661</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
    QSlider::add-page:Horizontal
     {     
background-color: rgba(255, 255, 255,100);
        height:4px;  
     } 
     QSlider::sub-page:Horizontal   
    {  
background-color: rgb(255, 255, 255);
        height:4px;
     }
    QSlider::groove:Horizontal   
    { 
        background:transparent; 
        height:4px;
    }  
    QSlider::handle:Horizontal  
    { 
        height:16px; 
        width:16px; 
	
	border-image: url(:/image/image/circle1.png);
  margin: -6 0px; 
    }
      
</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>20</y>
      <width>341</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
background-color: rgba(255, 255, 255,150);
border-style:outset;

border-width:0px;

border-radius:10px;
border-color: rgba(0,0,0,100);
font:bold 14px;
color:rgb(180,180,180);
padding:6px;
}</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>0</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
border-radius:0px;
background-image: url(:/image/image/min.png);
background-color: rgba(255, 255, 255,0);
background-repeat:no-repeat;
border:none;
background-position:center center;

}

QPushButton:hover{



	background-color: rgb(255, 255, 255,100);


}
QPushButton:pressed{
	background-color: rgba(95, 95, 95,100);
border-color:rgba(255,255,255,30);
border-style:inset;
color:rgba(0,0,0,100);

}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>1132</x>
      <y>0</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
border-radius:0px;
background-image: url(:/image/image/max.png);
background-color: rgba(255, 255, 255,0);
background-repeat:no-repeat;
border:none;
background-position:center center;

}
QPushButton:hover{

background-color: rgba(230,0,0,0.5);
}
QPushButton:pressed{
background-color: rgba(95, 95, 95,100);
border-color:rgba(255,255,255,30);
border-style:inset;
color:rgba(0,0,0,100);

}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>1165</x>
      <y>0</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
border-radius:0px;
background-image: url(:/image/image/quit.png);
background-color: rgba(255, 255, 255,0);
background-repeat:no-repeat;
border:none;
background-position:center center;

}
QPushButton:hover{

background-color: rgba(230,0,0,0.5);
}
QPushButton:pressed{
background-color: rgba(95, 95, 95,100);
border-color:rgba(255,255,255,30);
border-style:inset;
color:rgba(0,0,0,100);

}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{

	font: 75 pt &quot;Comic Sans MS&quot;;
color:rgba(255,255,255,0.5);
   
 

}</string>
    </property>
    <property name="text">
     <string>MyMusic</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>1068</x>
      <y>0</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
border-radius:0px;
background-image: url(:/image/image/tr.png);
background-color: rgba(255, 255, 255,0);
background-repeat:no-repeat;
border:none;
background-position:center center;

}

QPushButton:hover{



	background-color: rgb(255, 255, 255,100);


}
QPushButton:pressed{
	background-color: rgba(95, 95, 95,100);
border-color:rgba(255,255,255,30);
border-style:inset;
color:rgba(0,0,0,100);

}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>-32</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
color: rgba(255, 255, 255,0);
background-image:url(:/image/image/image_normal/setting.png);
background-repeat:no-repeat;
background-position:center center;
border:none;

}

QPushButton:hover{



background-image: url(:/image/image/image_hover/setting_hover.png);
background-repeat:no-repeat;
background-position:center center;


}
QPushButton:pressed{
background-image: url(:/image/image/image_normal/setting.png);
background-repeat:no-repeat;
background-position:center center;

}

</string>
    </property>
    <property name="text">
     <string>push</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>968</x>
      <y>-32</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
color: rgba(255, 255, 255,0);
background-image:url(:/image/image/image_normal/theme.png);
background-repeat:no-repeat;
background-position:center center;
border:none;

}

QPushButton:hover{



background-image: url(:/image/image/image_hover/theme_hover.png);
background-repeat:no-repeat;
background-position:center center;


}
QPushButton:pressed{
background-image: url(:/image/image/image_normal/theme.png);
background-repeat:no-repeat;
background-position:center center;

}

</string>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>916</x>
      <y>-32</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
color: rgba(255, 255, 255,0);
background-image:url(:/image/image/image_normal/add.png);
background-repeat:no-repeat;
background-position:center center;
border:none;

}

QPushButton:hover{



background-image: url(:/image/image/image_hover/add_hover.png);
background-repeat:no-repeat;
background-position:center center;


}
QPushButton:pressed{
background-image: url(:/image/image/image_normal/add.png);
background-repeat:no-repeat;
background-position:center center;

}

</string>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>864</x>
      <y>-32</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
color: rgba(255, 255, 255,0);
background-image:url(:/image/image/image_normal/quit2.png);
background-repeat:no-repeat;
background-position:center center;
border:none;

}

QPushButton:hover{



background-image: url(:/image/image/image_hover/quit2_hover.png);
background-repeat:no-repeat;
background-position:center center;


}
QPushButton:pressed{
background-image: url(:/image/image/image_normal/quit2.png);
background-repeat:no-repeat;
background-position:center center;

}

</string>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>640</y>
      <width>48</width>
      <height>48</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background-image: url(:/image/image/image_normal/pre.png);
background-repeat:no-repeat;
background-position:center center;
border:none;

}

QPushButton:hover{

	background-image: url(:/image/image/image_hover/pre_hover.png);
background-repeat:no-repeat;
background-position:center center;


}
QPushButton:pressed{
background-image: url(:/image/image/image_normal/pre.png);
background-repeat:no-repeat;
background-position:center center;

}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_10">
    <property name="geometry">
     <rect>
      <x>125</x>
      <y>640</y>
      <width>48</width>
      <height>48</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background-repeat:no-repeat;
background-position:center center;
border:none;

}

QPushButton:hover{
background-repeat:no-repeat;
background-position:center center;


}
QPushButton:pressed{
background-repeat:no-repeat;
background-position:center center;

}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_11">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>640</y>
      <width>48</width>
      <height>48</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background-image: url(:/image/image/image_normal/next.png);
background-repeat:no-repeat;
background-position:center center;
border:none;

}

QPushButton:hover{
background-image: url(:/image/image/image_hover/next_hover.png);
background-repeat:no-repeat;
background-position:center center;


}
QPushButton:pressed{
background-image: url(:/image/image/image_normal/next.png);
background-repeat:no-repeat;
background-position:center center;

}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>630</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>WhatsThisCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 italic 11pt &quot;Comic Sans MS&quot;;
color: rgb(112, 112, 112);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>110</y>
      <width>350</width>
      <height>531</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QListWidget" name="listWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>350</width>
        <height>531</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QListWidget{
font: 75 12pt &quot;Comic Sans MS&quot;;
background-color:transparent;

}
QListWidget::item{
color:rgb(120,120,120);
border:none;
margin-top:3px;

}
QListWidget::item:hover{
padding-left:4px;
margin-top:3px;
color:rgb(100,100,100);
background-color:rgba(255,255,255,0.2);



}
QListWidget::item:selected{
outline:none;
margin-top:3px;
padding-left:4px;
background-color:rgba(255,255,255,0.5);
color:rgb(0,0,0);
}
QListWidget::item:!active{
margin-top:3px;
border:none;
background-color:rgba(255,255,255,0);
color:rgb(120,120,120);
}



</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QListWidget" name="listWidget_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>350</width>
        <height>531</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QListWidget{
font: 75 12pt &quot;Comic Sans MS&quot;;
background-color:transparent;

}
QListWidget::item{
color:rgb(120,120,120);
border:none;
margin-top:3px;

}
QListWidget::item:hover{
padding-left:4px;
margin-top:3px;
color:rgb(100,100,100);
background-color:rgba(255,255,255,0.2);



}
QListWidget::item:selected{
outline:none;
margin-top:3px;
padding-left:4px;
background-color:rgba(255,255,255,0.5);
color:rgb(0,0,0);
}
QListWidget::item:!active{
margin-top:3px;
border:none;
background-color:rgba(255,255,255,0);
color:rgb(120,120,120);
}



</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QListWidget" name="listWidget_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>350</width>
        <height>531</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QListWidget{
font: 75 12pt &quot;Comic Sans MS&quot;;
background-color:transparent;

}
QListWidget::item{
color:rgb(120,120,120);
border:none;
margin-top:3px;

}
QListWidget::item:hover{
padding-left:4px;
margin-top:3px;
color:rgb(100,100,100);
background-color:rgba(255,255,255,0.2);



}
QListWidget::item:selected{
outline:none;
margin-top:3px;
padding-left:4px;
background-color:rgba(255,255,255,0.5);
color:rgb(0,0,0);
}
QListWidget::item:!active{
margin-top:3px;
border:none;
background-color:rgba(255,255,255,0);
color:rgb(120,120,120);
}



</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_4">
     <widget class="QListWidget" name="listWidget_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>350</width>
        <height>531</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QListWidget{
font: 75 12pt &quot;Comic Sans MS&quot;;
background-color:transparent;

}
QListWidget::item{
color:rgb(120,120,120);
border:none;
margin-top:3px;

}
QListWidget::item:hover{
padding-left:4px;
margin-top:3px;
color:rgb(100,100,100);
background-color:rgba(255,255,255,0.2);



}
QListWidget::item:selected{
outline:none;
margin-top:3px;
padding-left:4px;
background-color:rgba(255,255,255,0.5);
color:rgb(0,0,0);
}
QListWidget::item:!active{
margin-top:3px;
border:none;
background-color:rgba(255,255,255,0);
color:rgb(120,120,120);
}



</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_5">
     <widget class="QListWidget" name="listWidget_5">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>350</width>
        <height>531</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QListWidget{
font: 75 12pt &quot;Comic Sans MS&quot;;
background-color:transparent;

}
QListWidget::item{
color:rgb(120,120,120);
border:none;
margin-top:3px;

}
QListWidget::item:hover{
padding-left:4px;
margin-top:3px;
color:rgb(100,100,100);
background-color:rgba(255,255,255,0.2);



}
QListWidget::item:selected{
outline:none;
margin-top:3px;
padding-left:4px;
background-color:rgba(255,255,255,0.5);
color:rgb(0,0,0);
}
QListWidget::item:!active{
margin-top:3px;
border:none;
background-color:rgba(255,255,255,0);
color:rgb(120,120,120);
}



</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_12">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background-image: url(:/image/image/image_normal/music.png);
background-repeat:no-repeat;
background-position:center center;
border:none;

}

QPushButton:hover{

	background-image: url(:/image/image/image_hover/music_hover.png);
background-repeat:no-repeat;
background-position:center center;


}
QPushButton:pressed{
background-image: url(:/image/image/image_normal/music.png);
background-repeat:no-repeat;
background-position:center center;

}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_13">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>60</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background-image: url(:/image/image/image_normal/like.png);
background-repeat:no-repeat;
background-position:center center;
border:none;

}

QPushButton:hover{

	background-image: url(:/image/image/image_hover/like_hover.png);
background-repeat:no-repeat;
background-position:center center;


}
QPushButton:pressed{
background-image: url(:/image/image/image_normal/like.png);
background-repeat:no-repeat;
background-position:center center;

}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_14">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>60</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background-image: url(:/image/image/image_normal/list.png);
background-repeat:no-repeat;
background-position:center center;
border:none;

}

QPushButton:hover{

	background-image: url(:/image/image/image_hover/list_hover.png);
background-repeat:no-repeat;
background-position:center center;


}
QPushButton:pressed{
background-image: url(:/image/image/image_normal/list.png);
background-repeat:no-repeat;
background-position:center center;

}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_15">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>60</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background-image: url(:/image/image/image_normal/play-resently.png);
background-repeat:no-repeat;
background-position:center center;
border:none;

}

QPushButton:hover{

	background-image: url(:/image/image/image_hover/play-resently_hover.png);
background-repeat:no-repeat;
background-position:center center;


}
QPushButton:pressed{
background-image: url(:/image/image/image_normal/play-resently.png);
background-repeat:no-repeat;
background-position:center center;

}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_16">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>60</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background-repeat:no-repeat;
background-position:center center;
border:none;

}

QPushButton:hover{
background-repeat:no-repeat;
background-position:center center;


}
QPushButton:pressed{
background-repeat:no-repeat;
background-position:center center;

}
</string>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>600</y>
      <width>521</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 11pt &quot;Comic Sans MS&quot;;
color: rgb(112, 112, 112);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_17">
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>648</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{

background-repeat:no-repeat;
background-position:center center;
border:none;

}

QPushButton:hover{


background-repeat:no-repeat;
background-position:center center;


}
QPushButton:pressed{

background-repeat:no-repeat;
background-position:center center;

}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSlider" name="verticalSlider">
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>460</y>
      <width>35</width>
      <height>181</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider 
    {  
        background-color: rgba(255,255,255,0.2);
        min-width:5px; 
        max-width:5px;
        border:15px solid rgba(255,255,255,0.2); 
    }
     QSlider::add-page:vertical
     {      
       
	    background-color: rgb(250, 250, 0);
        width:4px;
     }
     QSlider::sub-page:vertical  
    {
        background-color: rgba(255,255,255,0.5); 
        width:4px;
     }
    QSlider::groove:vertical 
    {
        background:transparent;
        width:6px;
    }
    QSlider::handle:vertical 
    {
         height: 13px;
        width:13px;
 
 margin: -0 -4px; 
	border-image: url(:/image/image/circle_10.png);
     }</string>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_18">
    <property name="geometry">
     <rect>
      <x>1024</x>
      <y>648</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background-repeat:no-repeat;
background-position:center center;
border:none;

}

QPushButton:hover{
background-repeat:no-repeat;
background-position:center center;


}
QPushButton:pressed{
background-repeat:no-repeat;
background-position:center center;

}
</string>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
